test/cpp/end2end/ssl_credentials_test.cc

```diff
@@ -77,96 +129,297 @@ class SslCredentialsTest : public ::testing::Test {
     }
   }
 
+  absl::StatusOr<std::shared_ptr<const AuthContext>> DoRpc(
+      const SslCredentialsOptions& options, grpc_ssl_session_cache* cache,
+      bool override_ssl_target_name = true) {
+    ChannelArguments channel_args;
+    if (GetParam().use_session_cache) {
+      channel_args.SetPointer(std::string(GRPC_SSL_SESSION_CACHE_ARG), cache);
+    }
+    if (override_ssl_target_name) {
+      channel_args.SetSslTargetNameOverride("foo.test.google.fr");
```

Comment:
nit: consider pulling this magic value into a named constant for clarity

Reply:
Done.